src/libraries/Common/tests/TestUtilities/System/AssertExtensions.cs

```diff
@@ -678,6 +678,16 @@ public static E Throws<E, T>(string expectedParamName, Span<T> span, AssertThrow
             return exception;
         }
 
+        public static void FalseExpression(bool expr, [CallerArgumentExpression(nameof(expr))] string exprString = null)
+        {
+            Assert.False(expr, $"Expected \"false\" from the expression: \"{exprString}\".");
```

Comment:
I like the idea.  Have you verified that the failure message looks good in context?

Reply:
Yep. For example:
<img width="516" alt="Screenshot 2025-03-31 at 1 10 14 PM" src="https://github.com/user-attachments/assets/ba4a45da-ea74-4f2d-8290-5c99b9f8f61e" />